src/App.svelte

```diff
@@ -37,23 +36,27 @@
     </p>
   </div>
 
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/76584243/111971513-6edc1c80-8b22-11eb-83df-277a0a8f4fd8.png"
+      alt="" />
```

Comment:
```suggestion
      alt="EddieHub Banner" />
```
Maybe add some alternative text here

Reply:
Done!